Lib/glyphsLib/interpolation.py

```diff
@@ -79,24 +84,89 @@ def build_designspace(masters, master_dir, out_dir, instance_data):
     return writer.path, instance_files
 
 
-def add_masters_to_writer(writer, ufos):
+# TODO: Use AxisDescriptor from fonttools once designSpaceDocument has been
+# made part of fonttools. https://github.com/fonttools/fonttools/issues/911
+# https://github.com/LettError/designSpaceDocument#axisdescriptor-object
+AxisDescriptor = namedtuple('AxisDescriptor', [
+    'minimum', 'maximum', 'default', 'name', 'tag', 'labelNames', 'map'])
+
+
+def get_axes(masters, instances):
+    # According to Georg Seifert, Glyphs 3 will have a better model
+    # for describing variation axes.  The plan is to store the axis
+    # information globally in the Glyphs file. In addition to actual
+    # variation axes, this new structure will probably also contain
+    # stylistic information for design axes that are not variable but
+    # should still be stored into the OpenType STAT table.
+    #
+    # We currently take the minima and maxima from the instances, and
+    # have hard-coded the default value for each axis.  We could be
```

Comment:
The default should come from whatever master gets its `info` attribute set to True.  @brawer can you do that?

In the future, Georg said he'll add a custom property for base master.